src/main/java/org/sciborgs1155/robot/Hopper/HopperIO.java

```diff
@@ -0,0 +1,10 @@
+package org.sciborgs1155.robot.Hopper;
+
+import monologue.Logged;
+
+public interface HopperIO extends Logged {
```

Comment:
this extends logged -- is that purposeful? Do you, for instance, want to be logging the getSpeed method? If so:
a. add an @ Log annotation to log it
b. add an @ Log.File annotation in Hopper.java to log the HopperIO